### ISSUE TITLE
Connector 3.0 HowTo

### ISSUE THREAD

**Issue by Dimas-sc** [DHTMLX/connector-php#9]
Hi,
Until version 2.x I used the php connector with the DHTMLX Scheduler including these files on my CI controller:
- scheduler_connector.php
- db_phpci.php

With 3.0 I see the files have been reorganized and renamed. Which files I have to include now?

Thx

**Comment by EgorLazarovich** [comment]
Hello! Do you want to use connector for CI 3 via composer?

**Comment by Dimas-sc** [comment]
No, I'm downloading it manually and including it in the CI 3 controller with:

require_once("assets/dhtmlx/dhtmlxConnector/codebase/scheduler_connector.php");
require_once("assets/dhtmlx/dhtmlxConnector/codebase/db_phpci.php");

**Comment by Dimas-sc** [comment]
I need to change the way I've integrated the DHTMLX Connector with CI 3 controllers?

**Comment by baihaqyaviq** [comment]
Use Dhtmlx\Connector;

// to function
      $this->conn = new Connector\JSONDataConnector($this->db, "MySQLi");
        $this->conn->configure("events", "event_id", "start_date, end_date, event_name");
        $this->conn->useModel(Event);
        $this->conn->dynamic_loading(30);
        $this->conn->render();
